src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -0,0 +1,447 @@
+| ReconMethodParameterUnit             | REQUIRED              | [array][] of [strings][]               | Unit of reconstruction parameters (for example, `["none", "none"]`).                                            |
+| ReconMethodParameterValues           | REQUIRED              | [array][] of [numbers][]               | Values of reconstruction parameters (for example, `[21, 3]`)                                                    |
+| ReconFilterType                      | REQUIRED              | [string][] or [array][] of [strings][] | Type of post-recon smoothing (for example, `["Shepp"]`)                                                         |
+| ReconFilterSize                      | REQUIRED              | [number][] or [array][] of [numbers][] | Kernel size of post-recon filter (FWHM).                                                                        |
+| AttenuationCorrection                | REQUIRED              | [string][]                             | Short description of the attenuation correction method used.                                                    |
+| ReconMethodImplementationVersion     | RECOMMENDED           | [string][]                             | Identification for the software used, such as name and version.                                                 |
+| AttenuationCorrectionMethodReference | RECOMMENDED           | [string][]                             | Reference paper for the attenuation correction method used.                                                     |
+| ScaleFactor                          | RECOMMENDED           | [array][] of [numbers][]               | Scale factor for each frame.                                                                                    |
+| ScatterFraction                      | RECOMMENDED           | [array][] of [numbers][]               | Scatter fraction for each frame.                                                                                |
+| DecayCorrectionFactor                | RECOMMENDED           | [array][] of [numbers][]               | Decay correction factor for each frame.                                                                         |
+| PromptRate                           | RECOMMENDED           | [array][] of [numbers][]               | Prompt rate for each frame.                                                                                     |
+| RandomRate                           | RECOMMENDED           | [array][] of [numbers][]               | Random rate for each frame.                                                                                     |
+| SinglesRate                          | RECOMMENDED           | [array][] of [numbers][]               | Singles rate for each frame.                                                                                    |
```

Comment:
```suggestion
| SinglesRate                          | RECOMMENDED           | [array][] of [numbers][]               | Singles rate for each frame (same units as `Unit´, e.g. ´"Bq/mL"´).                                                                                    |
```

Reply:
One Markdown hint for you @mnoergaard: For code formatting you MUST use backticks, not "forwardticks". 

Looking at your suggestion, you use:
```
`Unit´
```

Which renders as:
`Unit´

You want to use two backticks, like:
```
`Unit´
```

Which makes it renders as:
`Unit`

You can go back to your suggestions and click "edit" to remedy these things, see screenshot:

![image](https://user-images.githubusercontent.com/9084751/107789549-03987100-6d52-11eb-8798-61731f4fa5e7.png)

Reply:
Thanks for pointing this out. I don't now why this has happened, as I usually use backticks - and I can't even reproduce the symbol (i.e. forwardticks) on my keyboard... Nevertheless, I have updated my previous suggestions :-)